app/decorators/visit_decorator.rb

```diff
@@ -45,6 +45,23 @@ def processed_at
     @processed_at ||= last_visit_state_change&.created_at || object.created_at
   end
 
+  def contact_list_working?
```

Comment:
No, this was wip as it dependend of yesterday's discussion. Coming soon with final logic